src/test/resources/dev-test/basic-dev-project/build.gradle

```diff
@@ -12,6 +12,9 @@ buildscript {
     repositories {
         mavenLocal()
         mavenCentral()
+        maven {
+            url "https://oss.sonatype.org/content/repositories/snapshots/com/ibm/websphere/appmod/tools/"
```

Comment:
Does this need the entire directory path?

Reply:
Updated to just "https://oss.sonatype.org/content/repositories"